crates/sys/src/lib.rs

```diff
@@ -5,14 +5,14 @@
 
 // Features that identify thumbv7em-none-eabihf.
 // Until target_abi is stable, this also permits thumbv7em-none-eabi.
-#[cfg(not(all(
+#[cfg(not(any(all(
     target_arch = "arm",
     target_feature = "thumb2",
     target_feature = "v7",
     target_feature = "dsp",
     target_os = "none",
     //target_abi = "eabihf",
-)))]
+), miri)))]
```

Comment:
Minor nitpicking: I would suggest identing different levels of the epression to make it easier to understand (if Clippy is fine with this):

```suggestion
[cfg(not(any(
    all(
        target_arch = "arm",
        target_feature = "thumb2",
        target_feature = "v7",
        target_feature = "dsp",
        target_os = "none",
        //target_abi = "eabihf",
    ),
    miri
)))]
```

otherwise it is hard to understand to which clause `miri` relates